hid-kye.c

```diff
@@ -483,6 +483,91 @@ static __u8 pensketch_m912_rdesc_fixed[] = {
+	0x95, 0x07,         /*      Report Count (7),               */
+	0xB1, 0x02,         /*      Feature (Variable),             */
+	0xC0,               /*  End Collection,                     */
+	0x05, 0x0D,         /*  Usage Page (Digitizer),             */
+	0x09, 0x02,         /*  Usage (Pen),                        */
+	0xA1, 0x01,         /*  Collection (Application),           */
+	0x85, 0x10,         /*      Report ID (16),                 */
+	0x09, 0x20,         /*      Usage (Stylus),                 */
+	0xA0,               /*      Collection (Physical),          */
+	0x09, 0x42,         /*          Usage (Tip Switch),         */
+	0x09, 0x44,         /*          Usage (Barrel Switch),      */
+	0x09, 0x46,         /*          Usage (Tablet Pick),        */
+	0x14,               /*          Logical Minimum (0),        */
+	0x25, 0x01,         /*          Logical Maximum (1),        */
+	0x75, 0x01,         /*          Report Size (1),            */
+	0x95, 0x03,         /*          Report Count (3),           */
+	0x81, 0x02,         /*          Input (Variable),           */
+	0x95, 0x04,         /*          Report Count (4),           */
+	0x81, 0x03,         /*          Input (Constant, Variable), */
+	0x09, 0x32,         /*          Usage (In Range),           */
+	0x14,               /*          Logical Minimum (0),        */
+	0x25, 0x01,         /*          Logical Maximum (1),        */
+	0x95, 0x01,         /*          Report Count (1),           */
+	0x81, 0x02,         /*          Input (Variable),           */
+	0x05, 0x01,         /*          Usage Page (Desktop),       */
+	0x09, 0x30,         /*          Usage (X),                  */
+	0x75, 0x10,         /*          Report Size (16),           */
+	0x95, 0x01,         /*          Report Count (1),           */
+	0xA4,               /*          Push,                       */
+	0x55, 0x0D,         /*          Unit Exponent (13),         */
+	0x65, 0x33,         /*          Unit (Inch^3),              */
+	0x14,               /*          Logical Minimum (0),        */
+	0x26, 0x00, 0x3C,   /*          Logical Maximum (15360),    */
+	0x34,               /*          Physical Minimum (0),       */
+	0x46, 0x00, 0x3C,   /*          Physical Maximum (15360),   */
+	0x81, 0x02,         /*          Input (Variable),           */
+	0x09, 0x31,         /*          Usage (Y),                  */
+	0x26, 0x00, 0x28,   /*          Logical Maximum (10240),    */
+	0x46, 0x00, 0x28,   /*          Physical Maximum (10240),   */
+	0x81, 0x02,         /*          Input (Variable),           */
+	0x05, 0x0D,         /*          Usage Page (Digitizer),     */
+	0x09, 0x30,         /*          Usage (Tip Pressure),       */
+	0x14,               /*          Logical Minimum (0),        */
+	0x26, 0xFF, 0x03,   /*          Logical Maximum (1023),     */
+	0x34,               /*          Physical Minimum (0),       */
+	0x46, 0xFF, 0x03,   /*          Physical Maximum (1023),    */
```

Comment:
Having Physical Maximum without units doesn't make sense. As nobody actually cares about pressure units, they're not consistent, and everyone just uses pressure curves, it's better to just remove these.